src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngProvider.cs

```diff
@@ -86,6 +86,14 @@ public override string ToString()
         // Well known NCrypt KSPs
         //
 
+        public static CngProvider MicrosoftPlatformCryptoProvider
```

Comment:
@DavidSimner, this will need to be added to the reference assembly as well:
https://github.com/dotnet/runtime/blob/54018cf9c836c2df358ccdc6ee5e3597bd270bb6/src/libraries/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs#L226-L227

Reply:
Done!

Reply:
I think the bot only adds the label when there is a modified ref file on PR creation. If a ref file is added later, the bot does not add the label. Is this correct, @maryamariyan ?